### DISCUSSION TITLE
Is it possible to write crossed-out math?

### DISCUSSION THREAD

**Discussion by dekkab** [ForNeVeR/wpf-math#348] [Q&A] [answered]
# Hello! Is it possible to do as here:
![image](https://user-images.githubusercontent.com/59896217/215277146-824a5de0-67c8-459c-85cb-6d93fde03b1e.png)

## [Source](https://tex.stackexchange.com/questions/75525/how-to-write-crossed-out-math-in-latex)

**Comment by ForNeVeR** [comment, marked as answer]
Not yet. We'll need to figure out from which package does this come in normal LaTeX distributions, and think about how to implement it.

I have opened a feature request #349 about this.

**Comment by dekkab** [comment]
Thanks
